experimental/PropertyDDS/packages/property-properties/src/propertyFactory.js

```diff
@@ -3,6 +3,8 @@
  * Licensed under the MIT License.
  */
 
+/* eslint-disable import/order */
```

Comment:
Any particular reason we can't enforce this here? If so, probably worth a comment here. In general, I think disable pragmas should always be accompanied by a note explaining why they are needed.

Reply:
The code seems to imply that the import order is intentional; I added a comment to clarify.